### DISCUSSION TITLE
`git log` inside a checkout step shows just single commit

### DISCUSSION THREAD

**Discussion by shreyanshp** [community/community#25652] [Actions]
<p>When using git log in action/checkout step, even after using fetch and pull, I can only see a single commit in github actions, whereas on my local (even you can check on your local, as I print the SHAs value in <code>git log fromBase..toRecent</code> values</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/shreyanshp/gitops/runs/3378787728?check_suite_focus=true">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/shreyanshp/gitops/runs/3378787728?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/21307950/181092365-5948085e-cfa4-41b7-996a-c7fff12840d0.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/shreyanshp/gitops/runs/3378787728?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

  <p>GitHub is where people build software. More than 65 million people use GitHub to discover, fork, and contribute to over 200 million projects.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment, marked as answer]
<p>Look up the <code>fetch-depth</code> parameter in the <a href="https://github.com/actions/checkout/blob/afe4af09a72596f47d806ee5f8b2674ec07fdc73/README.md">actions/checkout documentation</a>. By default it is 1, meaning only the latest commit is fetched.</p>

**Comment by shreyanshp** [comment]
<p>Yup, thank you so much!</p>
